system/Commands/Database/ShowTableInfo.php

```diff
@@ -0,0 +1,241 @@
+            CLI::newLine();
+
+            return;
+        }
+
+        if (array_key_exists('show', $params)) {
+            $this->showAllTables($tables);
+
+            return;
+        }
+
+        $tableName       = $params[0] ?? null;
+        $limitRows       = (int) ($params['limit-rows'] ?? 10);
+        $limitFieldValue = (int) ($params['limit-field-value'] ?? 15);
+
+        if (! in_array($tableName, $tables, true)) {
+            $tableNameNo = CLI::promptByKey(
+                ['Here is the list of your database tables:', 'Which table do you want to see?'],
+                $tables,
+                'required'
+            );
+
+            $tableName = $tables[$tableNameNo];
+        }
+
+        if (array_key_exists('metadata', $params)) {
+            $this->showFieldMetaData($tableName);
+
+            return;
+        }
+
+        $this->showDataOfTable($tableName, $limitRows, $limitFieldValue);
+    }
+
+    private function showDataOfTable(string $tableName, int $limitRows, int $limitFieldValue)
+    {
+        CLI::newLine();
+        CLI::write("Data of Table \"{$tableName}\":", 'black', 'yellow');
+        CLI::newLine();
+
+        $thead       = $this->db->getFieldNames($tableName);
+        $this->tbody = $this->makeTableRows($tableName, $limitRows, $limitFieldValue);
+        CLI::table($this->tbody, $thead);
+    }
+
+    private function showAllTables(array $tables)
+    {
+        CLI::write('The following is a list of the names of all database tables:', 'black', 'yellow');
+        CLI::newLine();
+
+        $thead       = ['ID', 'Table Name', 'Num of Rows', 'Num of Fields'];
+        $this->tbody = $this->makeTbodyForShowAllTables($tables);
+
+        CLI::table($this->tbody, $thead);
+        CLI::newLine();
+    }
+
+    private function makeTbodyForShowAllTables(array $tables): array
+    {
+        foreach ($tables  as $id => $tableName) {
+            $table = $this->db->protectIdentifiers($tableName);
+            $db    = $this->db->query("SELECT * FROM {$table}");
+
+            $this->tbody[] = [
+                $id + 1,
+                $tableName,
+                $db->getNumRows(),
+                $db->getFieldCount(),
+            ];
+        }
+
+        if ($this->sortDesc) {
+            krsort($this->tbody);
+        }
+
+        return $this->tbody;
+    }
+
+    private function makeTableRows(string $tableName, int $limitRows, int $limitFieldValue): array
+    {
+        $this->tbody = [];
+
+        $builder = $this->db->table($tableName);
+        $rows    = $builder->limit($limitRows)->get()->getResultArray();
+
+        foreach ($rows as $row) {
+            $row = array_map(
+                static function ($item) use ($limitFieldValue) {
+                    if (mb_strlen((string) $item) > $limitFieldValue) {
+                        return mb_substr((string) $item, 0, $limitFieldValue) . '...';
+                    }
+
+                    return $item;
+                },
```

Comment:
use arrow function should be doabe with ternary:

```suggestion
                static fn ($item): string => mb_strlen((string) $item) > $limitFieldValue) 
                    ? mb_substr((string) $item, 0, $limitFieldValue) . '...'
                    : $item
                ,
```

Reply:
@samsonasik 
> PHP Parse error:  syntax error, unexpected token "," in system/Commands/Database/ShowTableInfo.php on line 211